assets/app/views/_overview-deployment.html

```diff
@@ -138,7 +139,48 @@
         </div>
 
         <!-- spacer -->
-        <div flex class="visible-xs-block"></div>
+        <div flex></div>
+      </div>
+      <div ng-if="hpa.length" class="hpa-details">
+        Autoscaled:
+        <span class="nowrap">min: {{hpa[0].spec.minReplicas || 1}},</span>
+        <span class="nowrap">
+          max: {{hpa[0].spec.maxReplicas}}
+          <span
```

Comment:
Currently the messages are different for different types, and we add links to set values, etc. We'd lose some things.